stdlib/contextlib.pyi

```diff
@@ -103,9 +101,9 @@ if sys.version_info >= (3, 7):
         def push_async_exit(self, exit: _ACM_EF) -> _ACM_EF: ...
         def callback(self, callback: Callable[..., Any], *args: Any, **kwds: Any) -> Callable[..., Any]: ...
         def push_async_callback(self, callback: _CallbackCoroFunc, *args: Any, **kwds: Any) -> _CallbackCoroFunc: ...
-        def pop_all(self: _S) -> _S: ...
+        def pop_all(self: Self) -> Self: ...
```

Comment:
This method is on BaseExitStack

Reply:
The purpose of `_BaseExitStack` is to avoid copy/pasta in the implementation. On the one hand, we don't need it in typeshed because the stubs are short anyway, but on the other hand, it is generally good if stubs match the implementation. Either way, adding `_BaseExitStack` is beyond the scope of this PR.

Reply:
This will need to be fixed on both copies of pop_all instead